scripts/trivy_scan.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+
+set -e
+
+AWS_ACCOUNT_ID=$(aws sts get-caller-identity --output text --query Account)
+ECR_REGISTRY="$AWS_ACCOUNT_ID.dkr.ecr.$AWS_DEFAULT_REGION.amazonaws.com"
+IMAGE="$ECR_REGISTRY/$INPUT_ECR_REPOSITORY:$IMAGE_VERSION"
+S3_BUCKET_NAME=trivy-ops
+
+credentials=$(aws sts assume-role --role-arn arn:aws:iam::108141096600:role/ops-github-runner --role-session-name ops-s3)
+
+export AWS_PAGER=""
+aws configure set aws_access_key_id "$(echo "$credentials" | jq -r '.Credentials.AccessKeyId')" --profile ops
+aws configure set aws_secret_access_key "$(echo "$credentials" | jq -r '.Credentials.SecretAccessKey')" --profile ops
+aws configure set aws_session_token "$(echo "$credentials" | jq -r '.Credentials.SessionToken')" --profile ops
+
+echo "Print repo name: $GITHUB_REPOSITORY"
+echo "Download root trivy file from s3" 
+eval "aws --profile ops s3 cp s3://${S3_BUCKET_NAME}/.trivyignore ."
+
+PATH_TO_FOLDER=$GITHUB_REPOSITORY
+totalFoundObjects="$(aws s3 --profile ops ls s3://"${S3_BUCKET_NAME}"/"${PATH_TO_FOLDER}"/.trivyignore --recursive --summarize | grep "Total Objects:" | sed 's/[^0-9]*//g')"
```

Comment:
Refactor this and use `aws cp`. If `aws cp` fails then you know that there is no file like that in the s3 bucket.

Reply:
Any issue with aws s3 listing . If we have to go with fail command approach , again we have to unset error (like set +e out properties  and  enable it back . If there is any pressing reason , i also have changes ready for cp command fail approach .

Reply:
I have changed to aws cp though.